isamples_metadata/GEOMETransformer.py

```diff
@@ -147,7 +152,30 @@ def sample_description(self) -> str:
     def has_context_categories(self) -> list[VocabularyTerm]:
         # TODO: resolve https://github.com/isamplesorg/isamples_inabox/issues/312
         # This should probably return the biological kingdom once that is hooked into the vocabulary
-        return [vocabulary_mapper.sampled_feature_type().term_for_key("sf:marinewaterbody")]
+        kingdom = None
+        ranks = ["kingdom", "phylum", "genus"]
+        ranks_to_check = []
+        for rank in ranks:
+            value = self._source_record_main_record().get(rank)
+            if value is not None and value != "unidentified":
+                ranks_to_check.append(value)
+        for rank in ranks_to_check:
+            kingdom = self._taxonomy_name_to_kingdom_map.get(rank)
+            if kingdom is not None:
+                break
+        if kingdom is None:
+            informal_classification = self.informal_classification()
+            if len(informal_classification) > 0:
+                kingdom = self._taxonomy_name_to_kingdom_map.get(informal_classification[0])
+        if kingdom is not None:
+            if kingdom == "Chromista":
+                # Make a fake Chromista until the real one shows up (https://github.com/isamplesorg/vocabularies/issues/17)
```

Comment:
This is obviously temporary.  @smrgeoinfo not sure how long it takes to add/publish a new term?  If it will happen soon I can remove this hack.